include/rocksdb/db.h

```diff
@@ -1046,6 +1046,19 @@ class DB {
     return IngestExternalFile(DefaultColumnFamily(), external_files, options);
   }
 
+  // CreateColumnFamilyWithImport() will create a new column family with
+  // column_family_name and import external SST files specified in metadata into
+  // this column family.
+  // (1) External SST files can be created using SstFileWriter.
+  // (2) External SST files can be exported from a particular column family in
+  //     an existing DB.
+  // External files are deleted on a successful return.
+  virtual Status CreateColumnFamilyWithImport(
+      const ColumnFamilyOptions& options, const std::string& column_family_name,
+      const ImportColumnFamilyOptions& import_options,
+      const std::vector<LiveFileMetaData>& metadata,
+      ColumnFamilyHandle** handle) = 0;
```

Comment:
It doesn't have to be included here but is there a way we support import the data to default column family, assuming it is empty? This feels an interesting idea too.

Reply:
Thanks for the feedback. Will update and push against the master in next few days.

Reply:
Yes. This can be useful. Will handle this separately.